packages/gestalt/src/SelectList.js

```diff
@@ -119,11 +123,13 @@ function SelectList({
     <Box>
       {label && <FormLabel id={id} label={label} labelDisplay={labelDisplay} />}
       <Box
-        color={disabled ? 'secondary' : 'default'}
         display="flex"
         position="relative"
         rounding={4}
         width="100%"
+        dangerouslySetInlineStyle={{
```

Comment:
Why is the override needed?

Reply:
What was it before?

Reply:
It's the actual color for tokens. 

Before it was just default color from box which isn't the same as the other form fields. They were different in dark mode. Different gray.

I had to move the background color below the icon to surface it. The icon cannot go above as it affects the hovering behavior and the pointer.